src/SFA.DAS.AANHub.Application/Apprentices/Queries/GetApprenticeMemberResult.cs

```diff
@@ -7,6 +7,8 @@ public class GetApprenticeMemberResult
         public Guid MemberId { get; set; }
         public string? Email { get; set; }
         public string? Name { get; set; }
```

Comment:
I think we're trying to get away from nullable strings where not needed. These properties will always be populated if the Apprentice object is not null

Probably should  updated the string props to:
```suggestion
        public string Name { get; set; } = null!
```